src/lib.js

```diff
@@ -193,31 +193,43 @@ var curlConverter = {
         return arr.join("&");
     },
 
+    sanitizeCurlArgs: function(curlString) {
```

Comment:
this should be renamed with just sanitizeArgs or similar name. This is for sanitizing any bash string and not the specific curlString as discussed earlier. Hence the input argument name should also be changed to bash string ?